src/System.Runtime.Caching/src/System.Runtime.Caching.csproj

```diff
@@ -69,6 +73,9 @@
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
```

Comment:
You need to include MemoryMonitor.Unix.cs here as well.

Reply:
There is no MemoryMonitor.Unix.cs - only .Windows.cs version which contains a static constructor where we initialize  s_totalPhysical. Just for symmetry purpose we could include a pretty much empty MemoryMonitor.Unix.cs with an empty static constructor (to keep it static) but this is not really necessary (at least for right now while there is nothing else to initialize on Unix version).

Reply:
Oh you are right, I thought I had seen the file. Thanks for explaining.